markdown/events/2022/bytesize-resources-to-learn-nextflow.md

```diff
@@ -23,3 +23,59 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[14:18](https://www.youtube.com/watch?v=oO7rAp-QZOkt=858)
+The next learning resource is GitHub. The GitHub review process, it not only makes sure that the community coding standards are there too, but also it's a great way of, again, learning the whole process. As you have experts from the nf-core core team guiding you on making sure all the standards are met too and also giving you unique solutions. Interaction through GitHub is also an important part. As you are interacting with the nf-core community, just pooring over the existing GitHub repos of nf-core pipelines and looking at the processes are great places to learn as you will start building your own pipelines. There's more to collaboration than you think. One of the resources that I would recommend is the Turing Way guide for collaboration. Especially if you are new to GitHub and would like to know more about how the review process works and all that you can find more information through this guide. At this point, you should be quite capable of doing any of these things especially.
+
+[15:49](https://www.youtube.com/watch?v=oO7rAp-QZOkt=949)
+One is to run the nf-core pipelines straight out of the box. Especially if you're a facility and there's an nf-core pipeline that fits your needs. One is to use the nf-core pipeline straight out of the box. Second is to run the nf-core pipelines with some modifications. Either you turn off some of the features or you add some features or modules on your own with some modifications, or create an entirely new pipeline from scratch using the nf-core template. Now using the nf-core template gives the advantage of making sure that they all have the same structure as the other nf-core pipelines. You can then adapt a lot of pieces, especially modules and subworkflows from other nf-core pipelines into your templated pipeline that you're creating. In that case, all you have to do will be to basically shop for modules in nf-core if you have all the modules available there or consider making one if it isn't there, so that you are giving back to the community as well. And in some cases, you'll have to keep some local modules.
```

Comment:
```suggestion
One is to run the nf-core pipelines straight out of the box. Especially if you're a facility and there's an nf-core pipeline that fits your needs. One is to use the nf-core pipeline straight out of the box. Second is to run the nf-core pipelines with some modifications. Either you turn off some of the features or you add some features or modules on your own with some modifications, or create an entirely new pipeline from scratch using the nf-core template. Now using the nf-core template gives the advantage of making sure that they all have the same structure as the other nf-core pipelines. You can then adapt a lot of pieces, especially modules and subworkflows from other nf-core pipelines into your templated pipeline that you're creating. In that case, all you have to do will be to basically shop for modules in nf-core if you have all the modules available there or consider making one if it isn't there, so that you are giving back to the community as well. And in some cases, you'll have to keep some local modules.
```